hdl/fpga/platforms/bladerf-micro/software/bladeRF_nios/Makefile

```diff
@@ -2,23 +2,78 @@
 #              VARIABLES TO BE DEFINED BY PLATFORM MAKEFILE
 #------------------------------------------------------------------------------
 
-FPGA_COMMON_DIR    := ../../../../../../fpga_common
-LIBBLADERF_DIR     := ../../../../../../host/libraries/libbladeRF
-BLADERF_COMMON_DIR := ../../../common/bladerf/software/bladeRF_nios
+# paths are relative to hdl/fpga/platforms/bladerf-micro/software/bladeRF_nios
 
-QUARTUS_WORKDIR    := ../../../../../quartus/${WORKDIR}
-NIOS_BUILD_OUTDIR  := $(QUARTUS_WORKDIR)/bladeRF_nios
-MEM_INIT_DIR       := $(NIOS_BUILD_OUTDIR)/mem_init
-BSP_BUILD_OUTDIR   := $(QUARTUS_WORKDIR)/bladeRF_nios_bsp
-QUARTUS_OUTDIR     := $(QUARTUS_WORKDIR)/output_files
+FPGA_COMMON_DIR		:= ../../../../../../fpga_common
```

Comment:
Looks like most (all?) spaces were replaced by tabs. Was that intentional?

Reply:
that is an old habit for Makefiles and honestly it hasn't been necessary in... uh... decades.  will fix :)